### PR TITLE
Update codepush.gradle: fix Could not find matching constructor for: …

### PR DESCRIPTION
**Pull request by hamdij0maa**
./gradlew clean
fix Could not find matching constructor for java.io.File(File). Fixes : #2648

### CODE REVIEW

File: android/codepush.gradle [hunk 1]
```diff
@@ -60,8 +60,8 @@ gradle.projectsEvaluated {
             jsBundleDir = reactBundleTask.property('jsBundleDir').asFile.get()
             resourcesDir = reactBundleTask.property('resourcesDir').asFile.get()
             
-            new File(jsBundleDir).mkdirs()
-            new File(resourcesDir).mkdirs()
+            new File(jsBundleDir.toString()).mkdirs()
+            new File(resourcesDir.toString()).mkdirs()
```
Comment (mizinin) [inline android/codepush.gradle:64]:
```suggestion
            jsBundleDir.mkdirs()
            resourcesDir.mkdirs()
```
Cause jsBundleDir and resourcesDir are already instances of File

**Comment by GSTJ** [comment]
Needing this for https://github.com/GSTJ/react-native-code-push-plugin/issues/4

**Comment by mgscreativa** [comment]
Hi1 I confirm that after applying this PR my app builds ok on android and RN 73

**Comment by neutrino2211** [comment]
I manually made the proposed changes in a RN 72 project and it seems to work. Is there anything holding this back from being pushed to NPM? It's delaying my team's ability to compile new APKs

**Comment by BogdaKusch** [comment]
> I manually made the proposed changes in a RN 72 project and it seems to work. Is there anything holding this back from being pushed to NPM? It's delaying my team's ability to compile new APKs

you can use patch-package for now 😄

**Review by DmitriyKirakosyan — APPROVED** [review]

**Comment by GSTJ** [comment]
Captura de Tela 2024-01-23 às 11 45 50

Something's up with CI.

**Comment by AnatolyPristensky** [comment]
> ¿Is it possible to do a new release so this change gets published to npmJS?

Hello, @mgscreativa, this release is already on npmJS - https://www.npmjs.com/package/react-native-code-push

**Pull request merged**